uiowa/uiowa
# Feature: card bug fixes

- This pr resolves https://github.com/uiowa/uiowa/issues/3221 and resolves https://github.com/uiowa/uiowa/issues/3090. 
- Also includes a teaser image update for academic groups that was missed.  

# How to test

`blt ds --site=admissions.uiowa.edu`
`blt ds --site=sandbox.uiowa.edu`
`blt ds --site=grad.uiowa.edu`
`blt ds --site=grad.uiowa.edu`
`blt ds --site=ighn.international.uiowa.edu`
`yarn workspace uids_base gulp --development`
`drush @sandbox.local cr`
`drush @admissions.local cr`

1. Go to https://admissions.local.drupal.uiowa.edu/academics and verify that the card paragraphs further down on the page have margins between paragraphs. 
2. Go to https://sandbox.local.drupal.uiowa.edu/events-listing and compare description text size against https://sandbox.prod.drupal.uiowa.edu/events-listing - both should be similar. 
3. Go to https://sandbox.local.drupal.uiowa.edu/people-directory and compare description text size against https://sandbox.prod.drupal.uiowa.edu/people-directory - both should be similar. 
4. Go to https://sandbox.prod.drupal.uiowa.edu/aggregator-examples and compare description text size against https://sandbox.local.drupal.uiowa.edu/aggregator-examples - both should be similar. 
5. Go to http://sandbox.local.drupal.uiowa.edu/cards and verify that the button has a margin between the card text and button like on http://sandbox.prod.drupal.uiowa.edu/cards.   Also test removing the card text and verify that the empty `<div class="card__description">` does not render. 
6. Also verify that button has space between card title and bttn if no card text is present. 

Also test:

- https://grad.local.drupal.uiowa.edu/academics/thesis-defenses/list
- https://grad.local.drupal.uiowa.edu/dei/srop/mentors
- https://grad.local.drupal.uiowa.edu/dei/srop/past-scholars/2019
- https://internationalighn.local.drupal.uiowa.edu/
- https://grad.local.drupal.uiowa.edu/grad-success/3MT/2020-3mt-showcase
